app/presenters/policies_list_presenter.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class PoliciesListPresenter
+  def initialize(policies)
+    @policies = policies
+  end
+
+  def get_registry
+    return nil unless @policies
```

Comment:
It should never be in real execution time, the only time it can be nil is from the tests for this line:
https://github.com/3scale/porta/blob/6336b1d300bdf28252b500ff294c550fc6514e60/test/integration/api/policies_controller_test.rb#L10
But because it is being mocked without returning anything, which cannot happen in real life because if it does not return what we need it is because there has been an error raised and then it does not even reach here anyway.
I think that we should adapt the test to the real code instead of type checking unnecessarily

Reply:
Fair enough